src/Draco.Debugger/IO/IoWorker.cs

```diff
@@ -43,36 +43,34 @@ public IoWorker(TProcess process, RemoteIoHandles handles)
     /// </summary>
     /// <param name="cancellationToken">Can be used to cancel the worker.</param>
     /// <returns>The task of the worker loop.</returns>
-    public Task Run(CancellationToken cancellationToken) => Task.Run(async () =>
+    public async Task Run(CancellationToken cancellationToken) => await Task.WhenAll(
```

Comment:
From what I understand, a `Run` method that return a `Task` doesn't make sense.  
By the API alone, I would understand I can Run multiple time the IO Worker.  
But since it use a pair of stream passed in constructor, I think this class should have the ownership of the Task.  
IMO the IOWorker should have a synchronous Start, and a Stop method that may be called one day when running cleanup code.